### ISSUE TITLE
It is not possible to open folder with keyboard in the left bar

### ISSUE THREAD

**Issue by NN---** [ForkIssues/TrackerWin#540]
Up and Down keys move the focus but for branches with slashes a/b it is not possible to expand when the top name is folded.

**Comment by sindrijo** [comment]
Yes, would be nice if you could collapse these "folders" with the arrow keys.
